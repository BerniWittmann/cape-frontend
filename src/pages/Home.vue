<template>
  <v-layout>
    <div class="home">
      <img alt="CaPE logo" src="../assets/logo.png">
      <h3>{{ $t('home.greeting') }}</h3>
      <p>{{ $t('home.construction_notice') }}</p>
    </div>
  </v-layout>
</template>

<script>
// @ is an alias to /src
import DefaultLayout from '@/layouts/Default.vue'

export default {
  components: {
    VLayout: DefaultLayout
  }
}
</script>

<style scoped lang="scss">
.home {
  text-align: center;
}
</style>
