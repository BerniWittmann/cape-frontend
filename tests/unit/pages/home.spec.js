import { shallowMount } from '@vue/test-utils'
import { i18n } from '../setupPlugins'

import Home from '@/pages/Home.vue'

import EmptySlotComponent from '../EmptySlotComponent.vue'

describe('Pages', () => {
  describe('Home.vue', () => {
    it('renders', () => {
      const wrapper = shallowMount(Home, {
        i18n,
        stubs: {
          'v-layout': EmptySlotComponent
        }
      })
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
