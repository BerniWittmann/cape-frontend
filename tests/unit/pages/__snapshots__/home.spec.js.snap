// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Home.vue renders 1`] = `
<div>
  <div class="home"><img alt="CaPE logo" src="../assets/logo.png">
    <h3>home.greeting</h3>
    <p>home.construction_notice</p>
  </div>
</div>
`;
